docs/source/dataframe-parquet.rst

```diff
@@ -0,0 +1,247 @@
+Dask Dataframe and Parquet
+==========================
+
+.. currentmodule:: dask.dataframe
+
+Parquet_ is a popular, columnar file format designed for efficient data storage
+and retrieval. Dask dataframe includes :func:`read_parquet` and
+:func:`to_parquet` functions/methods for reading and writing parquet files
+respectively. Here we document these methods, and provide some tips and best
+practices.
+
+Reading Parquet Files
+---------------------
+
+.. autosummary::
+    read_parquet
+
+Dask dataframe provides a :func:`read_parquet` function for reading parquet
+files and datasets. Its first argument is one of:
```

Comment:
Maybe add a short section describing the difference between a parquet file and a parquet dataset? It could even be a sentence, like "... parquet files or datasets (a directory of parquet files with the same structure)"